frontend/packages/console-shared/src/components/dashboard/utilization-card/UtilizationItem.tsx

```diff
@@ -115,7 +125,12 @@ export const UtilizationItem: React.FC<UtilizationItemProps> = React.memo(
     setTimestamps,
   }) => {
     const { t } = useTranslation();
-    const { data, chartStyle } = mapLimitsRequests(utilization, limit, requested);
+    const { data, chartStyle } = mapLimitsRequests(
+      utilization,
+      limit,
+      requested,
+      trimSecondsXMutator,
```

Comment:
@rebeccaalpert this is probably the fix you want for pod CPU/Memory charts tooltips

Reply:
I didn't seem to have an effect, but thanks for the the advice!